falco.yaml

```
@@ -1055,6 +1055,7 @@ metrics:
   state_counters_enabled: true
   kernel_event_counters_enabled: true
   libbpf_stats_enabled: true
+  plugins_enabled: true
```

Assuming this is a `[stable]` feature.

---

```suggestion
  plugins_custom_metrics_enabled: true
```
Just to make it more clear, WDYT?

---

or `plugins_metrics_enabled` if you feel it's too long.

---

@mrgian would you mind to also add a new comment to the description. Sharing a draft below, but feel free to adjust it:

```
# `plugins_custom_metrics_enabled`: Falco can now expose your custom plugins' 
# metrics. Please note that if the respective plugin has no metrics implemented, 
# there will be no metrics available. In other words, there are no default or 
# generic plugin metrics at this time. This may be subject to change.
```